t/extract-version.t

```diff
@@ -648,20 +648,20 @@ foreach my $test_case (@modules) {
     # We want to ensure we preserve the original, as long as it's legal, so we
     # explicitly check the stringified form.
     {
-      local $TODO = !defined($got) && ($test_case->{TODO_code_sub} || $test_case->{TODO_scalar});
+      local $TODO = !defined($got) && ($test_case->{TODO_code_sub} || $test_case->{TODO_scalar}) || undef;
```

Comment:
This is more strict than the original, but if the change is consequential it will result in more tests failing due to a lack of a `TODO_*` where there should be one, so it should make the problem more apparent. (That is, I think this change is not harmful.)